### PR TITLE
Add multimedia test description

### PR DESCRIPTION
**Pull request by ylatuya**

### CODE REVIEW

File: engineering/multimedia/README.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+When the user drags the slider it will change the position of the stream with a seek.
+The final design including the slider should like like:
+
+`Position: HH:mm:ss.SSS [-----------------|---------------] Duration: HH:mm:ss.SSS`
+
+The story is successful when the user starts playing a clip and the slider is updated with the relative position.
+The story is successful when the drags the slider and a seek is performed updating the postion of the stream.
+
+### Create timeline with a snapshots preview
+As a user I would like to visually preview the contents of the stream once its loaded, in the form of 10 thumbnails displayed in a timeline. Thumbnails are spaciated in time at 1/10th of the duration of the clip. The thumnails generation should happen in background and the timeline should be updated asynchonously with the new thumbnails as they are generated.
```
Comment (dabrain34) [inline engineering/multimedia/README.md]: thumbnails instead of thumnails

File: engineering/multimedia/README.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+ * Whatever other thing you want to show us!
+
+## The rules
+
+We will provide you with a description of a new feature for the video player app, as it would be done by our Product Manager, and a list of User Stories describing its functionality.
+You will create the new application and publish your work in a public repository using a service like github or bitbucket.
+As you will see, the User Stories do not have any UI design, so design it as you want since it won’t be taken in account.
+What we will value instead, is the ability to work with new API’s, your autonomy to interpret and implement User Stories, the use of good design patterns and your ability to split your work into commits and branches.
+
+![hacker](https://raw.githubusercontent.com/fluendo/recruitment/master/content/hacker1.gif)
```
Comment (dabrain34) [inline engineering/multimedia/README.md]: for me gif is too agressive

File: engineering/multimedia/README.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+  * videoplayer-gtk3: sample video player using the Gtk+3 framework
+  * snapshot: sample application to get snapshots from a video file
+  
+The sources of the video player are taken from the GStreamer project examples and tutorials with the intention to provide a very basic starting point to start implementing new features for the test.
+
+### Setup
+On a machine with a Debian-like OS, install the following packages:
+
+```
+sudo apt-get install cmake libqt4-dev qtgstreamer-plugins libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev libboost-dev libqtgstreamer-dev libgles2-mesa-dev gstreamer1.0-libav libgdk-pixbuf2.0-dev gtk-3-dev
```
Comment (dabrain34) [inline engineering/multimedia/README.md]: gtk-3-dev does not exist -> libgtk-3-dev

File: engineering/multimedia/README.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+As a user, I would to seek to a position in the stream in the video player using a slider.
+Considering the slider has a range between 0 and 1, the value of the slider is the relative position of the stream, calculated like `current_position/duration`.
+The position of the slider will be updated with the same interval as the position label.
+When the user drags the slider it will change the position of the stream with a seek.
+The final design including the slider should like like:
+
+`Position: HH:mm:ss.SSS [-----------------|---------------] Duration: HH:mm:ss.SSS`
+
+The story is successful when the user starts playing a clip and the slider is updated with the relative position.
+The story is successful when the drags the slider and a seek is performed updating the postion of the stream.
```
Comment (dabrain34) [inline engineering/multimedia/README.md]: position

File: engineering/multimedia/README.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+make
+```
+
+### Run
+To run the app launch:
+```
+./videoplayer
+```
+
+The player starts without any video file openned. Click the open button and select a video file.
```
Comment (dabrain34) [inline engineering/multimedia/README.md]: open according to https://dictionary.cambridge.org/grammar/british-grammar/open-or-opened

File: engineering/multimedia/README.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+On a machine with a Debian-like OS, install the following packages:
+
+```
+sudo apt-get install cmake libqt4-dev qtgstreamer-plugins libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev libboost-dev libqtgstreamer-dev libgles2-mesa-dev gstreamer1.0-libav libgdk-pixbuf2.0-dev gtk-3-dev
+```
+
+### Configure
+The project uses a CMake build system. To configure it enter either videoplayer-qt or videoplayer-gtk3 depending on the toolkit of your choice and configure it with:
+```
+cmake CMakeLists.txt
```
Comment (dabrain34) [inline engineering/multimedia/README.md]: None of the folders/projects can be configured, missing dependencies on ubuntu 16.04. Is it on purpose ?

Comment (ylatuya) [reply engineering/multimedia/README.md]: They should all be configurable. What's missing ?

Comment (dabrain34) [reply engineering/multimedia/README.md]: my mistake, I was in cerbero shell ...

**Pull request merged**